app/src/main/java/com/techcourse/handler/HandlerAdapterRegistry.java

```diff
@@ -0,0 +1,27 @@
+package com.techcourse.handler;
+
+import com.interface21.webmvc.servlet.HandlerAdapter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HandlerAdapterRegistry {
+
+    private final List<HandlerAdapter> handlerAdapters;
+
+    public HandlerAdapterRegistry() {
+        this.handlerAdapters = new ArrayList<>();
+    }
+
```

Comment:
생성자에서 매개변수를 받아 초기화 작업을 하지 않고 객체를 생성한다면, 필드에서 초기화 하는 방법도 좋을 것 같습니다.

```suggestion
public class HandlerAdapterRegistry {

    private final List<HandlerAdapter> handlerAdapters = new ArrayList<>();

```